src/pages/Lend/LoanChannelCategoryExperiment.vue

```diff
@@ -1,5 +1,248 @@
+					image: this.imageRequire('./lbc-parents.jpg'),
+					loanChannelId: 26
+				},
+				'ending-soon': {
+					title: 'Ending soon',
+					tagline: 'Find time-sensitive loans across a variety of areas',
+					image: this.imageRequire('./lbc-endingsoon.jpg'),
+					loanChannelId: 3
+				},
+			};
+		},
+		targetedCategory() {
+			return {
+				// eslint-disable-next-line max-len
+				description: this.description,
+				...this.categories[this.targetedLoanChannel]
+			};
+		},
+		cssVars() {
+			return {
+				'--hero-background': `url(${this.targetedCategory.image})`
+			};
+		},
+		loans() {
+			return this.loanChannel?.loans?.values ?? [];
+		},
+		firstLoan() {
+			return [this.loans[0]];
+		},
+		remainingLoans() {
+			const [, ...rest] = this.loans;
+			return rest;
+		},
+		mgTargetCategory() {
+			const currentRoute = this.$route.path.replace('/lend-by-category/', '');
+			const targetRoutes = [
+				{ route: 'women', url: '/monthlygood?category=women', label: 'women' },
+				{ route: 'loans-to-women', url: '/monthlygood?category=women', label: 'women' },
+				{ route: 'education', url: '/monthlygood?category=education', label: 'students' },
+				{ route: 'loans-for-education', url: '/monthlygood?category=education', label: 'students' },
+				{ route: 'refugees-and-i-d-ps', url: '/monthlygood?category=refugees', label: 'refugees' },
+				{ route: 'loans-to-refugees-and-i-d-ps', url: '/monthlygood?category=refugees', label: 'refugees' },
+				{ route: 'eco-friendly', url: '/monthlygood?category=eco_friendly', label: 'eco-friendly loans' },
+				{ route: 'eco-friendly-loans', url: '/monthlygood?category=eco_friendly', label: 'eco-friendly loans' },
+				{ route: 'agriculture', url: '/monthlygood?category=agriculture', label: 'farmers' },
+				{ route: 'loans-to-farmers', url: '/monthlygood?category=agriculture', label: 'farmers' },
+				{ route: 'kiva-u-s', url: '/monthlygood?category=us_borrowers', label: 'U.S. borrowers' },
+				{ route: 'loans-to-u-s-small-businesses', url: '/monthlygood?category=us_borrowers', label: 'U.S. borrowers' }, // eslint-disable-line max-len
+				{ route: 'united-states-loans', url: '/monthlygood?category=us_borrowers', label: 'U.S. borrowers' },
+			];
+			return targetRoutes.find(route => route.route === currentRoute);
+		}
+	},
+	apollo: {
+		query: pageQuery,
+		preFetch: true,
+		result({ data }) {
+			const loanChannelValues = data?.lend?.loanChannels?.values ?? [];
+			this.isVisitor = !data?.my?.userAccount?.id;
+			// eslint-disable-next-line max-len
+			this.description = loanChannelValues.find(channel => channel.id === this.targetedCategory.loanChannelId).description;
+		},
+	},
+};
+</script>
+
+<style lang="postcss" scoped>
+.lbc-hero-wrapper {
+	@apply tw-bg-brand tw-bg-cover tw-bg-no-repeat tw-flex tw-w-full lg:tw-min-h-[320px] xl:tw-min-h-[460px];
+}
+
+/* Medium screens and above */
+@media screen and (min-width: 734px) {
+	.lbc-hero-wrapper {
+		@apply tw-bg-transparent tw-rounded-b-lg;
+
```

Comment:
Custom gradient css, tailwind doesnt have this fine grained control for gradients and multiple backgrounds.